addons/danger/XEH_preInitClient.sqf

```diff
@@ -36,11 +36,11 @@ _suppress_AI = {
 };
 
 // functions ~ easy hide
-_hide_AI = {
+private _fnc_hide_AI = {
     private _buildings = [player,38,true,true] call FUNC(findBuildings);
     private _units = (units player) select {_x distance player < 55 && {!isPlayer _x}};
     {
-        [_x,_pos,_range,_buildings] call FUNC(hideInside);
+        [_x, getPos _x, 45, _buildings] call FUNC(hideInside); // What should be the Default Distance here? @nkenny?
```

Comment:
Just set it to _nil_ or 10 meters. The default range needs revisiting in any case.

Reply:
3 lines about you checked in a distance of  38 units. and I added a bit more that is why I did use 45. but yeah 10 it then is

Reply:
Will probably need adjustment based to fit game play concerns.  It could have been a CBA setting-- but I don't want the bloat. And we're not a UI mod anyway.